pulp_rpm/app/serializers.py

```diff
@@ -338,8 +338,21 @@ class RpmPublicationSerializer(PublicationSerializer):
     A Serializer for RpmPublication.
     """
 
+    metadata_checksum_type = serializers.CharField(
+        help_text=_("The checksum for metadata."),
+        required=False,
+        allow_null=True,
+    )
+    package_checksum_type = serializers.CharField(
+        help_text=_("The checksum for packages."),
+        required=False,
```

Comment:
what about making this a choice field? in constants.py we have checksum_choices that can be used?
or we need to add a validation in the serializer so users cannot pass some invalid string that is not a valid checksum type.

Reply:
I'll check if createrepo_c supports all checksums types that are on checksum_choices

Reply:
@ipanova createrepo_c does support all checksum types, but I didn't know what to put as default on the `ChoiceField`